src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -161,32 +158,17 @@ def generate_initial_population(self, pop_data: SimulantData) -> None:
 
         self.population_view.update(pop)
 
-    def choose_standard_households(self, target_number_sims: int) -> pd.DataFrame:
-        standard_households = self.population_data["households"][
-            self.population_data["households"]["household_type"] == "Housing unit"
-        ]
-        # oversample households
-        chosen_households = vectorized_choice(
-            options=standard_households["census_household_id"],
-            n_to_choose=target_number_sims,
-            randomness_stream=self.randomness,
-            weights=standard_households["household_weight"],
-        )
-
-        # create unique id for resampled households
-        chosen_households = pd.DataFrame(
-            {
-                "census_household_id": chosen_households,
-                "household_id": np.arange(
-                    data_values.N_GROUP_QUARTER_TYPES,
-                    len(chosen_households) + data_values.N_GROUP_QUARTER_TYPES,
-                ),
-            }
-        )
+    def populate_standard_households(self) -> pd.DataFrame:
+        households = self.households.get_households()
+        target_number_sims = households.index.size
 
+        standard_households = households.loc[
+            households["census_household_id"] != "N/A",
```

Comment:
Why not use "housing_type" here?

Reply:
I have actually already fixed this in the next pr